spec/request/service_credential_bindings_spec.rb

```diff
@@ -1601,6 +1601,12 @@ def check_filtered_bindings(*bindings)
                                                                    'code' => 10_015
                                                                  }))
           end
+
+          it 'rolls back the transaction' do
+            api_call.call(admin_headers)
+
+            expect(service_instance.service_keys.count).to eq(0)
```

Comment:
Here you have to check `service_instance.service_bindings`. Service keys are always 0 even without the transaction.